packaging/firehol/firehol.functions

```diff
@@ -11,14 +11,15 @@ firehol_check_file() {
     sbin/Makefile.in)
       :
     ;;
-    configure.ac|sbin/commands.sed.in)
-      check_commands sbin/firehol.in || status=1
-      check_commands sbin/fireqos.in || status=1
-      check_commands sbin/link-balancer.in || status=1
-      check_commands sbin/update-ipsets.in || status=1
-      check_commands sbin/vnetbuild.in || status=1
+    configure.ac|sbin/install.config.in.in)
+      check_commands sbin/firehol || status=1
+      check_commands sbin/fireqos || status=1
+      check_commands sbin/link-balancer || status=1
+      check_commands sbin/update-ipsets || status=1
+      check_commands sbin/vnetbuild || status=1
```

Comment:
Required... well all they do is run checks at commit time, nothing in the actual code. I think they are still useful - only last week I wrote `rm -f whatever` newly in a script and these reminded me I should have used  `${RM_CMD}`.